### PR TITLE
docs: update cursor-based pagination usage on the README.md

### LINKED ISSUE #368: improve documentation regarding pagination

### PR DESCRIPTION
**Pull request by sborrazas**
Closes #368

### CODE REVIEW

**Review by thepiwo — APPROVED** [review]
thanks

File: README.md [hunk 1]
```diff
@@ -268,10 +268,9 @@ Querying for transactions via `txs` endpoint supports 3 kinds of parameters spec
 - generic ids
 - transaction fields
 
-Pagination supported via specifying of 2 parameters:
-
-- limit
-- page
+To be able to traverse through the list of transactions, the `next` field
+provided on the current page response is provided as explained in the
```
Comment (jyeshe) [inline README.md:272]: "the `next` field provided on the current page response is provided"... 
suggest ->
"the `next` field returned on the current page response is provided"...

File: README.md [hunk 2]
```diff
@@ -338,7 +337,7 @@ $ curl -s "https://mainnet.aeternity.io/mdw/txs/forward?type=channel_create&limi
       "tx_index": 87
     }
   ],
-  "next": "txs/gen/0-265258?limit=1&page=2&type=channel_create"
+  "next": "/txs/forward?cursor=73270&limit=1&type=channel_create"
```
Comment (jyeshe) [inline README.md:340]: :+1:

**Review by jyeshe — COMMENTED** [review]
small suggest

File: README.md (hunk 1 above)
Comment (sborrazas) [reply README.md]: done

**Review by jyeshe — APPROVED** [review]

**Pull request merged**